types/fast_node.go

```diff
@@ -29,13 +30,13 @@ func NewFastNode(key []byte, value []byte, version int64) *FastNode {
 
 // DeserializeFastNode constructs an *FastNode from an encoded byte slice.
 func DeserializeFastNode(key []byte, buf []byte) (*FastNode, error) {
```

Comment:
Should we move the regular `Node` to this package as well?

Reply:
It really feels like they belong together / grouped into the same package

Reply:
I think they should, just takes more refactors to get there